atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -353,7 +354,10 @@ def get_build(self, metadata, worker_metadatas):
 
         flatpak_source_info = get_flatpak_source_info(self.workflow)
         if flatpak_source_info is not None:
-            extra['image'].update(flatpak_source_info.koji_metadata())
+            compose_info = get_compose_info(self.workflow)
+            koji_metadata = compose_info.koji_metadata()
+            koji_metadata['flatpak'] = True
```

Comment:
Let's update the docs to list out this new koji metadata value.

Reply:
> Let's update the docs to list out this new koji metadata value.

It's not new with this patch - it just got moved around. So it's already in there: 

     - `build.extra.image.flatpak` (boolean): true if this image is a Flatpak